installation/rasppi.md

```diff
@@ -6,5 +6,87 @@ layout: documentation
 
 # Installation on a Raspberry Pi
 
-to be done...
+There’s no pre-configured image for OpenHAB, so installation is done the old fashioned way via a command line.
+
+Start with the latest (full) Raspbian SD image (not the “lite” version, these don’t include the Java Virtual Machine). Get your network cable plugged in, then boot up, and navigate through SSH. Run:
+
+`sudo raspi-config`
+
+Expand the filesystem; and from the advanced menu, change the memory split to 16. When are done, restart, and as good practice, run a full update
+
+`sudo apt-get update
+sudo apt-get upgrade`
+
+The easier way to install the OpenHAB runtime is via apt-get, but first we need to add a secure key and the new repository:
+
+`wget -qO - 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab' |sudo apt-key add -
```

Comment:
Does a single escaping work as expected for code blocks? Did you check the Jeykll-rendered output?